.travis.yml

```
@@ -124,6 +124,12 @@ install:
     fi
 
 before_script:
+  - |
+    # clone seqan2 into submodules
+    git clone https://github.com/seqan/seqan.git submodules/seqan
+    cd submodules/seqan
+    git checkout develop
+    cd ../../
   - mkdir ../seqan3-build
```

```suggestion
    # clone seqan2 into submodules
    git clone --depth 1 --branch develop https://github.com/seqan/seqan.git submodules/seqan
  - mkdir ../seqan3-build
```

Is shorter and probably faster (doesn't need to fetch the whole history)